components/inlinealert/stories/template.js

```diff
@@ -62,7 +63,9 @@ export const Template = ({
 
 		>
 			<div class="${rootClass}-header">
-				${variant.charAt(0).toUpperCase() + variant.slice(1)} ${headerText}
+				${when(headerText, () => 
+					headerText.charAt(0).toUpperCase() + headerText.slice(1)
+				)}
```

Comment:
```suggestion
				${headerText}
```

Since this is a string, we don't need a when wrapper and I'm not sure we should be altering the format of the string. Thanks for making this update!

Reply:
Oh and a note to double check the *.test.js file too - it looks like it lost the modifiers in the header text.

Reply:
Oh wait- did _those_ need that `${variant.charAt(0).toUpperCase() + variant.slice(1)}` in the test file instead of the template file? Is that what you mean as modifiers?